packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -676,6 +676,7 @@ export class AudiusAPIClient {
 
     this._assertInitialized()
 
+    // TODO: pass currentUserId
```

Comment:
can you add more context for this comment? assuming this if we eventually generate endpoints in track response yeah?

Reply:
Yeah this is a very temporary thing, Randy just made a change to pass currentUserId to /stream so this will be getting refactored after I do the backend changes today